app-muscadine-box/src/components/features/wallet/ConnectButton.tsx

```
@@ -1,49 +1,53 @@
-                onClick={() => open()}
-                variant="ghost"
-                size="md"
-                icon={<WalletIcon size="sm" />}
-                fullWidth
-                className="justify-start"
-            >
-                <span className="text-xs">{truncatedAddress}</span>
-            </Button>
-        );
-    }
+export default function ConnectButton({}: ConnectButtonProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)